src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ContainerTests.ps1

```diff
@@ -12,31 +12,61 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
+$resourceGroupName = "labRG1";
+$resourceName = "pstestrsvault";
+$vaultLocation = "westus";
```

Comment:
You should not use hard-coded locations, but get the locations by listing the provider locations, and preferring a location liek west us if it shows up in the list.  This makes it possible to run the tests against different azure environemnts, which may or may not have a particular location.